docs/bipoc.rst

```diff
@@ -1,29 +1,35 @@
 
-<!-- SOME STATS WOULD BE USEFUL HERE. STATS FROM STC? -->
+According to Open MIC's 2017 Report, `Breaking the Mold: Investing in Racial Diversity in Tech`_:
 
-To shrink this racial disparity and to build equity, we offer this space as BIPOC tech writers to:
+.. _`Breaking the Mold: Investing in Racial Diversity in Tech`: http://breakingthemold.openmic.org/
+
+* Black people, Latinos and Native Americans are underrepresented in tech by 16-to-18 percentage points compared with their presence in the U.S. labor force overall. Black people and Latinos each comprise just 5.3 percent of the Professionals category in U.S. tech industry labor data.
+* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
+* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.
+
+To shrink racial disparities and to build equity, we offer this space as BIPOC tech writers to:
 
 #. Emphasize the fact that we are already here, supporting one another and leading our communities.
 #. Encourage the professional development of BIPOC in tech writing, software development, and related fields.
 
 Purpose
 -------
 
-BIPOC need dedicated spaces for themselves in every profession, everywhere. We believe that one important way to practice anti-racism is to help other BIPOC communicators advance in their careers.
+BIPOC need dedicated spaces in every profession, everywhere. We believe that one important way to practice anti-racism is to help other BIPOC communicators advance in their careers.
 
-In a global field that is largely represented by `white`_ people, admonishing racism in spaces full of non-BIPOC people is limited in its impact. For one thing, these spaces do not capture the complexity of experiences that BIPOC have lived. BIPOC individuals and communities often experience various `intersections`_ of both privilege (e.g., `colorism`_) and discrimination (e.g., ability, gender identity, sexuality, citizenship status, neuro-status, linguistic `fluency`_).
+In a global field that is largely represented by `white`_ people, admonishing racism in spaces full of non-BIPOC people is limited in its impact. For one thing, these spaces do not capture the complexity of experiences that BIPOC have lived. BIPOC individuals and communities often experience various `intersections`_ of both privilege (e.g., `colorism`_) and discrimination (e.g., physical/cognitive ability, gender identity, sexuality, citizenship status, neuro-status, socioeconomic status, linguistic `fluency`_, religion, ethnicity).
 
 
-.. _white: https://nmaahc.si.edu/sites/default/files/styles/image_caption/public/images/captioned/whiteculture_info_1.png
```

Comment:
The internet's a terrible place, so the infographic that used to be at this link got co-opted by racists being like "this sounds great!" - so I changed the link to go to the page where that infographic used to be.

Reply:
Thank you so much!!!! Another friend pointed this out to me and I was wondering what had happened to it ...

As a side note: I actually saved that infographic to my computer a while back. We could probably use it on the page, but I think it's better and less obtrusive to use the link ref that you updated. 👍

Reply:
No problem, I'm glad I was able to catch it!

I was able to find the infographic on the Wayback Machine, so we could still make a hyperlink pretty easily through there if we wanted to. But I think the co-opting problem points to a greater issue with the infographic: if people are so indoctrinated to accept cultural white supremacy, they won't see anything wrong with what it says. So maybe it won't hurt to have that more basic information available in case people need it.